app/components/Headsign.tsx

```diff
@@ -5,9 +5,12 @@ function Headsign(props: { departure: any }) {
     const {headsign} = departure
 
     return (
+        headsign
+        ? (
         <div>
             {headsign}    
-        </div>
+        </div>)
```

Comment:
In this case the parent should have prevented the component from rendering

Reply:
i will remove the control.